accession-commons-jpa/src/test/resources/features/deprecation.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Accession deprecation
+
+  Scenario: I want to deprecate an object but the object does not exist.
+    When user deprecates unknown-accession reason: because i say so
```

Comment:
I know it was already like that, but let's not mix first and third person in acceptance tests. If the scenario says "I want to deprecate" then the other clauses should also use "I".

A better constructed phrase would be `When I deprecate unknown-accession because: I say so` (this also means the reason would be only `I say so`).

Reply:
i think use of 3rd person ( user ) would be more appropriate and also just needed to update the scenario headings .